packages/selinux-policy/processes.cil

```diff
@@ -15,14 +15,14 @@
 
 ; Permission group for relaxing security constraints on processes.
 (classmapping processes relax (
-  process (execheap execmem execstack)))
+  process (execheap execmem execstack noatsecure)))
 (classmapping processes relax (
   memprotect (mmap_zero)))
 
 ; Permission group for other actions.
 (classmapping processes interact (
   process (not (
     dyntransition transition setcurrent setexec
-    setfscreate setkeycreate setsockcreate
+    setfscreate setkeycreate setsockcreate noatsecure
```

Comment:
nit: this could go on the line with `execheap` instead - the permissions on each line reflect the entries in the earlier classmaps